src/Symfony/Component/Asset/Exception/AssetNotFoundException.php

```diff
@@ -19,10 +19,10 @@ class AssetNotFoundException extends RuntimeException
     private $alternatives;
 
     /**
-     * @param string     $message      Exception message to throw
-     * @param array      $alternatives List of similar defined names
-     * @param int        $code         Exception code
-     * @param \Throwable $previous     Previous exception used for the exception chaining
+     * @param string      $message      Exception message to throw
+     * @param array       $alternatives List of similar defined names
+     * @param int         $code         Exception code
+     * @param ?\Throwable $previous     Previous exception used for the exception chaining
```

Comment:
for consistency with other similar changes:
```suggestion
     * @param \Throwable|null $previous     Previous exception used for the exception chaining
```